packages/button/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "@constellation-components/button",
   "author": "maecapozzi <maecapozzi@gmail.com>",
-  "version": "1.0.1-alpha.5",
+  "version": "1.0.2",
   "description": "> TODO: description",
-  "main": "./dist/src/button.js",
-  "types": "./dist/src/button.d.ts",
+  "main": "./dist/button.js",
+  "types": "./dist/button.d.ts",
   "scripts": {
     "build": "tsc -p ./tsconfig.json"
   },
+  "files": [
+    "dist"
+  ],
```

Comment:
![image](https://user-images.githubusercontent.com/11462208/88431665-cf1a9300-cdc8-11ea-8644-a212f7bf3918.png)

Reply:
No more `src` directory in the build output